lib/tasks/va_facilities.rake

```diff
@@ -68,7 +68,228 @@ namespace :va_facilities do
         end
       end
     end
+    puts "All VA facilities have been created or updated in the DB!"
+  end
+
+
+  task :update_va_facilities => :environment do
+    ctr = 0
+    file = File.read("#{Rails.root}/lib/assets/va_facilities.json")
+    va_facilities = JSON.parse(file)
+    visns = Visn.all
+    visns.each do |visn|
+      puts 'VISN: ' +  visn.number.to_s
+      va_facilities.each do |vaf|
+          if visn.number === vaf["VISN"].to_i && VaFacility.where(station_number: vaf["Station_Number"]).empty?
+            puts 'Creating facility - ' + vaf["Official_Station_Name"]
+            hidden = false
```

Comment:
Do we need this, since we set the default to `false`?

Reply:
Ur right - we don't need it.